src/styles/class-overrides.scss

```diff
@@ -116,11 +122,15 @@
   border-bottom-style: solid;
   border-bottom-color: color('primary');
   &:hover {
+    text-decoration: none;
```

Comment:
Is this necessary if it's already in line 120?

Reply:
Without this line, the white underline still showed up on hover in addition to the bottom border